islands/IssuesList.tsx

```diff
@@ -0,0 +1,80 @@
+import { GitHubIssue } from '../lib/github.ts';
+import Card from '../components/Card.tsx';
+import { useState } from 'preact/hooks';
+
+type Props = {
+  repos: Array<string>;
+  issues: Array<GitHubIssue>;
+};
+
+type LabelProps = {
+  name: string;
+  color: string;
+}
+
+const IssuesList = ({repos, issues}: Props) => {
+  const [repoFilter, setRepoFilter] = useState(null);
+  const [labelFilter, setLabelFilter] = useState({});
+
+  const clearFilters = () => {
+    setRepoFilter(null);
+    setLabelFilter({});
+  }
+
+  const showLabel = (label: LabelProps) => {
+    return (
+      <span onClick={() => setLabelFilter(label)} style={"background-color: #".concat(label.color)} class="cursor-pointer font-bold mr-1 px-2 py-[0.125rem] rounded-xl text-white text-xxs whitespace-nowrap hover:text-black">
+        {label.name}
+      </span>
+    );
+  };
+
+  return (
+    <Card title="Current Issues">
+      <div class="flex justify-between my-4">
+        <div>
+          <select class="px-4 py-2" value={repoFilter} onChange={e => setRepoFilter(e.target.value)}>
+            <option value={null}>Filter by Repo</option>
+            {repos.map(repo => <option value={repo}>{repo}</option>)}
+          </select>
+          <span class="ml-4">
+            {labelFilter.name ? (
```

Comment:
```suggestion
            {labelFilter ? (
```

Since it's the whole `labelFilter` that can be `undefined` now.